component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -298,7 +298,7 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 	{
 		$strippedmatch = preg_replace('/(#|:|;)+[^}]*/', '', $matchesarray[0][$i]);
 
-		if (in_array($strippedmatch, $search))
+		if (in_array($strippedmatch, $search, FALSE))
```

Comment:
False is the default value for the third parameter so there is no need to add this.  See http://php.net/manual/en/function.in-array.php